scss/inc/_grid.scss

```diff
@@ -14,14 +14,17 @@
 }
 
 .grid-container {
-    overflow-x: hidden; 
+    overflow-x: hidden;
 }
 
-.grid-row, .fixed-grid-row, .colrow {
+.grid-row, .fixed-grid-row {
     @extend .clearfix;
     width: widthPerc(852, 840);
 }
-
+.colrow {
+    @extend .clearfix;
+    width: revert;

```

Comment:
Is the `width: revert` necessary? As far as I can see, `.colrow` is an isolated element not targeted by any other style rules, so leaving it with no width would give the default (`auto`). I checked in the browser and it will measure correctly:
![Screenshot 2022-09-12 at 13 57 35](https://user-images.githubusercontent.com/43652944/189647525-6ec12d14-1400-4105-930c-aaa6ec36b142.png)